example_scripts/spectrum_viewer.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python3
+
+from __future__ import print_function
```

Comment:
I don't think we need this anymore, we don't support python 2 anyways :)

Reply:
Old habits die hard... I will remove it